R/readQFeatures.R

```diff
@@ -46,13 +52,11 @@
 ##' input (see next section).
 ##'
 ##'
-##' ## Multi-set case
+##' @section Multi-set case:
 ##'
 ##' A run/batch variable is required to import multi-set data. It can
-##' either be defined as a `runCol` argument only, or a `runCol`
-##' argument and a `runCol` variable in `colData` (see multi-set cases
-##' in the example). These cases can be represented schematically as
-##' shown below.
+##' either be defined as a `runCol` argument only. These cases can be
+##' represented schematically as shown below.
```

Comment:
Why have you removed the part with `colData`? One of the confusing things is that the same behaviour can be obtained with different sets of parameters, so this needs to be explained.

Reply:
I guess you have seen that I moved the detailed documentation of the colData in a separate "section". I think because it may be confusing, it is better to first explain the concept with the schematics and introduce the `quantCols` and `runCol` arguments, and then introduce the `colData` argument separately.

Reply:
Of course, I can revert the paragraph structure and keep colData description as part of the use cases, but than I still think we should introduce colData after runCol and quantCols.